spec/models/department_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Department, type: :model do
+  before(:each) do
+    @department = create(:department)
+  end
+  describe "creation" do
+    it "should have no itme creation after being created" do
```

Comment:
it "creates a new object" do
  described_class.create(attributes_for(:department))
end